__tests__/get-article-by-id.test.js

```diff
@@ -0,0 +1,50 @@
+const request = require("supertest");
+
+const db = require("../db/connection");
+const {
+  articleData,
+  commentData,
+  topicData,
+  userData,
+} = require("../db/data/test-data");
+const seed = require("../db/seeds/seed");
+const app = require("../app");
+
+beforeEach(() => seed({ articleData, commentData, topicData, userData }));
+afterAll(() => db.end());
+
+describe("GET /api/articles/:article_id", () => {
+  test("200: that responds with the article with the given article id", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then(({ body }) => {
+        const { article } = body;
+        expect(article).toMatchObject({

```

Comment:
You need to check for the article_id on this object and make sure its the one you've requested. At the moment, this would pass for any article.

Reply:
Added another commit to address this.